cfme/tests/control/test_compliance.py

```diff
@@ -62,7 +62,9 @@ def compliance_vm(request, provider_key, provider_crud):
         appliance.configure(setup_fleece=True)
         vm = Vm(appliance.vm_name, provider_crud)
     # Do the final touches
-    with appliance.ipapp:
+    appliance.ipapp.browser_steal = True
+    with appliance.ipapp as appl:
+        appl.set_session_timeout(86400)
```

Comment:
IPAppliance does not automatically set browser_steal. Adding the argument would btw mean adding a __call__ method that would accept that parameter.